packages/stylelint-config-terra/stylelint.config.js

```diff
@@ -0,0 +1,48 @@
+const browserslist = require('@cerner/browserslist-config-terra');
+
+module.exports = {
+  extends: 'stylelint-config-sass-guidelines',
+  plugins: [
+    'stylelint-no-unsupported-browser-features',
+    './lib/rules/custom-property-no-duplicate-declaration/custom-property-no-duplicate-declaration',
+    './lib/rules/custom-property-name/custom-property-name',
+    './lib/rules/custom-property-pattern/custom-property-pattern',
+    './lib/rules/custom-property-pseudo-selectors/custom-property-pseudo-selectors',
+    './lib/rules/custom-property-namespace/custom-property-namespace',
+  ],
+  rules: {
+    'max-nesting-depth': 3,
+    'no-extra-semicolons': [true, { severity: 'warning' }],
+    'scss/at-mixin-pattern': '^[a-z]+([a-z0-9-]+[a-z0-9]+)?$',
+    'custom-property-pattern': [
+      '[a-z]+([a-z0-9-]+[a-z0-9]+)?$',
+      {
+        message: 'Custom property names should be written in lowercase with hyphens',
+      },
+    ],
+    'declaration-property-unit-whitelist': [{ 'line-height': [] }, { severity: 'warning' }],
+    'terra/custom-property-name': true,
+    'terra/custom-property-no-duplicate-declaration': true,
+    'terra/custom-property-pattern': true,
+    'terra/custom-property-pseudo-selectors': true,
+    'terra/custom-property-namespace': true,
+    'plugin/no-unsupported-browser-features': [
+      true,
+      {
+        browsers: browserslist,
```

Comment:
It looks like we can define the browsers explicitly or how browserlist recommends:

![Screen Shot 2020-12-10 at 9 01 51 AM](https://user-images.githubusercontent.com/14099737/101789096-7709d200-3ac6-11eb-9743-58df6e7ea37a.png)

Any chance did we test dropping this and the dependency on `@cerner/browserslist-config-terra`? One less dependency if it works and it would test the browsers specific to the project (if they don't match).

Reply:
It's also a recommendation: https://www.npmjs.com/package/stylelint-no-unsupported-browser-features#recommendations

Reply:
I think its a good idea to drop directly declaring the browserslist in favor of using the browserslist declared at root of whatever project.